bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -567,6 +567,15 @@ private function addObjectsNode(ArrayNodeDefinition $rootNode)
                     ->ignoreExtraKeys()
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->booleanNode('ignoreLocalizedQueryFallback')
```

Comment:
```suggestion
                        ->booleanNode('ignore_localized_query_fallback')
```
We should go for snake_case here, to be consistent and follow the recommendation of Symfony.